### PR TITLE
Installer and .js tweaks for gui on Windows

### PR DESCRIPTION
**Pull request by zanderz**
The important stuff is from @MarcoPolo.
@maxtaco

### CODE REVIEW

**Comment by MarcoPolo** [comment]
@keybase/react-hackers

**Comment by MarcoPolo** [comment]
Since I did it we should have someone else check it too

File: desktop/app/console-helper.js [hunk 1]
```diff
@@ -42,8 +42,8 @@ function tee (...writeFns) {
   return t => writeFns.forEach(w => w(t))
 }
 
-const stdErrWriter = t => process.stderr.write(t)
-const stdOutWriter = t => process.stdout.write(t)
+const stdErrWriter = t => process.platform !== 'win32' && process.stderr.write(t)
+const stdOutWriter = t => process.platform !== 'win32' && process.stdout.write(t)
```
Comment (MarcoPolo) [inline desktop/app/console-helper.js]: This means we won't see stdout logs even if we run it from the cmd line in windows. I'm okay with that for now, but maybe later we can see if we can write first, and then decide to log to stdout

File: desktop/app/console-helper.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ let fileWritable = null
 // If the file is too big, let's reset the log
 if (logFile) {
   fs.access(logFile, fs.W_OK, err => {
-    if (err && err.errno !== -2 /* file does not exist */) {
+    if (err && (err.errno !== -2 && err.errno !== -4058) /* file does not exist */) {
```
Comment (MarcoPolo) [inline desktop/app/console-helper.js]: This check should also be platform specific, but it's fine for now as a quick fix for windows

Comment (chrisnojima) [inline desktop/app/console-helper.js]: Why even have this check on errno at all? if (err) should be enough right?

File: desktop/app/console-helper.js (hunk 1 above)
Comment (chrisnojima) [inline desktop/app/console-helper.js]: maybe better to just do the check first so we don't call this and check every time (also can be optimized out with webpack) either 

```
const stdErrWriter = process.platform !== 'win32' ? t =>  && process.stderr.write(t) : () => {}
```

or change the call to tee()

**Comment by zanderz** [comment]
So does this mean it's ok to merge for now, with a jira story to revisit the issues mentioned?

**Comment by chrisnojima** [comment]
Let's actually fix this stuff. Won't take long

File: desktop/app/console-helper.js (hunk 2 above)
Comment (MarcoPolo) [inline desktop/app/console-helper.js]: If the error is that the file doesn't exist, we can fix that by creating the file. If the error is anything else, we fail

Comment (chrisnojima) [inline desktop/app/console-helper.js]: we can use the exists call right? just feels weird to use these magic errno

**Comment by MarcoPolo** [comment]
@chrisnojima tal plz

**Comment by chrisnojima** [comment]
:+1: could use fs.stat() to figure out if it exists or not and not use the errno stuff

**Pull request merged**